java/statestore/src/main/java/sleeper/statestore/transactionlog/snapshots/TransactionLogSnapshotDeleter.java

```diff
@@ -59,21 +59,26 @@ public TransactionLogSnapshotDeleter(
      * Searches for snapshots that have expired based on the current time, then deletes the snapshot file in addition to
      * the snapshot metadata in the metadata store.
      *
-     * @param currentTime the current time
+     * @param  currentTime the current time
+     * @return             the total number of snapshots attempted to be deleted
```

Comment:
This isn't quite correct, it will only return if all the snapshots were deleted sucessfully. It can just say "the total number of snapshots deleted".